lcf2xml/src/main.cpp

```diff
@@ -117,6 +136,26 @@ std::string GetPath(const std::string& str) {
 	return s;
 }
 
+FileCategories GetFilecategory(const std::string& in_file)
+{
+	std::ifstream in(in_file.c_str());
+
+	char buf[128];
+	memset(buf, '\0', 128);
```

Comment:
Any specific reason for `128`?

Reply:
Just saying, because `in.read(buf, 3);` does not read "some further bytes later"...